api/app/models/question.rb

```diff
@@ -28,4 +28,8 @@
 
 class Question < ApplicationModel
   attr_accessor :id, :text, :description, :default, :format, :ask_when, :template
+
+  def to_h
+    QuestionSerializer.new(self).attributes.merge({ 'id' => id }).deep_dup
+  end
```

Comment:
Is this used anywhere?

Reply:
cd91ba5a4814508182dc40a6903cb8d7c4af3f28